src/document.c

```diff
@@ -37,6 +39,37 @@ static void freeDocumentContext(void *p) {
   free(aCtx);
 }
 
+static void *newLatinTokenizerAlloc() {
+  return NewSimpleTokenizer(NULL, NULL, 0);
+}
+static void *newCnTokenizerAlloc() {
+  return NewChineseTokenizer(NULL, NULL, 0);
+}
+static void tokenizerFree(void *p) {
+  RSTokenizer *t = p;
+  t->Free(t);
+}
+
+static RSTokenizer *getTokenizer(const char *language, Stemmer *stemmer, StopWordList *stopwords) {
+  RSTokenizer *t;
+
+  if (language && strcasecmp(language, "chinese") == 0) {
```

Comment:
I suppose we better make sure this is usually called with NULL to avoid the strcasecmp